docs/docusaurus.config.js

```diff
@@ -42,7 +42,10 @@ const config = {
   },
 
   // Plugings need installing first then add here
-  plugins: ["docusaurus-plugin-sass"],
+  plugins: [
+    "docusaurus-plugin-sass",
+    require.resolve('docusaurus-lunr-search'),
```

Comment:
Seems like this package was last [updated 9 months ago](https://www.npmjs.com/package/docusaurus-lunr-search) and development seems to be dead, unlike this one: https://github.com/easyops-cn/docusaurus-search-local which is more active. 

If it's simple enough to use the other one, then I would recommend using that.

Reply:
@aktech — I tried it and it didn't work with our present setup last time.

Reply:
Sure, I don't mind using this one either as it's not mission critical, but can you describe the issue in a comment for posterity.